src/wavtools/lib/wav_recorder.js

```diff
@@ -251,54 +251,16 @@ export class WavRecorder {
    * @returns {Promise<true>}
    */
   async requestPermission() {
-    const ensureUserMediaAccess = async () => {
+    try {
+      console.log('ensureUserMediaAccess');
       const stream = await navigator.mediaDevices.getUserMedia({
```

Comment:
we only call this when listing devices (inside this module) .. and when connecting to the pipeline, so it feels like we should just call it without restriction.
We also now expose it through the API in case the user wants to force it.